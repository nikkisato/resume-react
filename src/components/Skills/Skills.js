import React from 'react';
import data from '../resume2';

export default function Skills() {
  return (
    <div>
      <p>
        <b>{data.Skills[0].Area}</b>
        <br></br>
        {data.Skills[0].SkillSet[0].Name}
        <br></br>
        {data.Skills[0].SkillSet[1].Name}
        <br></br>
        {data.Skills[0].SkillSet[2].Name}
        <br></br>
        {data.Skills[0].SkillSet[3].Name}
        <br></br>
        {data.Skills[0].SkillSet[4].Name}
        <br></br>
        {data.Skills[0].SkillSet[5].Name}
      </p>
    </div>
  );
}
