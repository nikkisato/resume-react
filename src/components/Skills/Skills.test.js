import React from 'react';
import { shallow } from 'enzyme';
import Skills from './Skills';

describe('Skills component', () => {
  it('renders Skills', () => {
    const wrapper = shallow(<Skills />);
    expect(wrapper).toMatchSnapshot();
  });
});
