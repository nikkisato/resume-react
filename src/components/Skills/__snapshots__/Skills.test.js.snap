// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills component renders Skills 1`] = `
<div>
  <p>
    <b>
      Tech Stack
    </b>
    <br />
    MongoDb
    <br />
    Node.js
    <br />
    Express
    <br />
    PostgreSQL/SQL
    <br />
    Q-Unit
    <br />
    Jest/Travis
  </p>
</div>
`;
